Samples/outlook-set-signature/webpack.config.js

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable no-undef */
+
+const devCerts = require("office-addin-dev-certs");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+
+const urlDev = "https://localhost:3000";
+//const urlProd = "https://www.contoso.com"; // CHANGE THIS TO YOUR PRODUCTION DEPLOYMENT LOCATION
+const urlProd = "https://tabbc9b35.z22.web.core.windows.net";
```

Comment:
I think the value of urlProd needs to change.

Reply:
fixed.